build_cmake/configs/base.cmake

```diff
@@ -145,10 +151,16 @@ config_bool(
     DEPENDS_ERROR ON "Failed to find tcmalloc library"
 )
 
+set(default_optimize_level)
+if("${WT_OS}" STREQUAL "windows")
+    set(default_optimize_level "/O2")
```

Comment:
I take it '/02' is MSVC's top level?

Reply:
Yep, this is the top level on Windows: 
* https://docs.microsoft.com/en-us/cpp/build/reference/o1-o2-minimize-size-maximize-speed?view=msvc-160
* https://docs.microsoft.com/en-us/cpp/build/reference/o-options-optimize-code?view=msvc-160

Reply:
👍